.circleci/config.yml

```diff
@@ -83,32 +83,61 @@ jobs:
 
 
 
-  test-11_check-van-mx:
```

Comment:
can we use the full word `vanilla` here? a shortening in `van` loses the context for what we are running. Please have a look at all jobs. I also see `worker` shortened to `work` and `isolation` to `iso`.

Unless there is a requirement in length of job names I'd rather have them spelled out fully